packages/core/src/Stores/client-store.js

```diff
@@ -347,6 +349,17 @@ export default class ClientStore extends BaseStore {
         return false;
     }
 
+    @computed
+    get is_eu() {
```

Comment:
please help to add TODO here to remove and resolve conflict for merging eu branch later